src/server/command_handler.py

```diff
@@ -17,17 +17,44 @@
 
 from src.utils.logging_manager import LoggingManager
 
+from src.parser.parser import Parser
+from src.optimizer.statement_to_opr_convertor import StatementToPlanConvertor
+
+from src.executor.plan_executor import PlanExecutor
+from src.optimizer.plan_generator import PlanGenerator
+from src.utils.logging_manager import LoggingManager
+
 
 @asyncio.coroutine
-def handle_request(transport, request_message):
+def handle_request(transport, query):
     """
         Reads a request from a client and processes it
 
         If user inputs 'quit' stops the event loop
         otherwise just echoes user input
     """
+    parser = Parser()
+    eva_statement = parser.parse(query)
+    insert_statement = eva_statement[0]
+
+    LoggingManager().log("Result from the parser: " + str(insert_statement))
+
+    convertor = StatementToPlanConvertor()
+    convertor.visit(insert_statement)
+
+    convertor.plan
+
+    eva_statement = parser.parse(query)
+    response_message = ""
+
+    for i in range(len(eva_statement)):
 
-    response_message = "foo"
+        LoggingManager().log("Parse Tree: " + str(eva_statement[i]))
+        logical_plan = StatementToPlanConvertor().visit(eva_statement[i])
```

Comment:
I feel this logic goes into a different file. We can call it a driver. 

According to me, the command handler should only be responsible for:
1.  instantiating the Driver code 
2. Take user queries as input and pass it to the driver
3. Take the output of the driver, and convert it into any format for displaying/ data transfer.

Command handler should only be an interface between User Interface layer and the Core Eva code. It should be lightweight.